admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+| `isActive`  |  yes | [Boolean](#boolean) | A flag to indicate whether or not the settlement model is active. |
+
+## API Resource `/transactions`
+
+The services provided by the resource `/transactions` are used by the Hub Operator for retrieving transfer details.
+
+### `GET /transactions/{id}`
+
+The `GET /transactions/{id}` request is used to retrieve information about a particular transaction.
+
+#### Path parameters
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `id`  |  yes | [UUID](#uuid) | Transfer identifier. |
+
+#### Data model
+
+The `GET /transactions/{id}` request sends an empty request body.
+
+The following data is returned in the case of a successful operation:
+
+| Name  | Required  | Type | Description |
+|---|---|--|--|
+| `quoteId`  |   | [UUID](#uuid) | Quote identifier. |
+| `transactionId`  |   | [UUID](#uuid) | Transaction identifier. |
+| `transactionRequestId`  |   | [String](#string) | Identifies an optional previously-sent transaction request. |
+| `payee`  |   | [Party](#party) | Payee details. |
+| `payer`  |   | [Party](#party) | Payer details. |
+| `amount`  |   | [Money](#money) | Transaction amount. |
+| `transactionType`  |   | [TransactionType](#transactiontype) | Transaction details. |
+| `note`  |   | [String](#string) | A memo that will be attached to the transaction. |
+| `extensionList`  |   | [ExtensionList](#extensionlist) | Additional details. |
+
+# Data models used by the API
+
+## Format
+
+For details on the formats used for element data types used by the API, see section [7.1 Format Introduction](../fspiop-api/documents/API-Definition_v1.1.md#71-format-introduction) in the Mojaloop FSPIOP API Definition.
+
+## Element data type formats
+
+This section defines element data types used by the API.
+
+### Amount
+
+For details, see section [7.2.13 Amount](../fspiop-api/documents/API-Definition_v1.1.md#7213-amount) in the Mojaloop FSPIOP API Definition.
+
+### Boolean
+
+A `"true"` or `"false"` value.
+
+### DateTime
+
+For details, see section [7.2.14 DateTime](../fspiop-api/documents/API-Definition_v1.1.md#7214-datetime) in the Mojaloop FSPIOP API Definition.
+
+### Enum
+
+For details, see section [7.2.2 Enum](../fspiop-api/documents/API-Definition_v1.1.md#722-enum) in the Mojaloop FSPIOP API Definition.
+
+### Integer
+
+For details, see section [7.2.5 Integer](../fspiop-api/documents/API-Definition_v1.1.md#725-integer) in the Mojaloop FSPIOP API Definition.
+
+### Number
+
+The API data type **Number** is a an arbitrary-precision, base-10 decimal number value.
+
+### String
+
+For details, see section [7.2.1 String](../fspiop-api/documents/API-Definition_v1.1.md#721-string) in the Mojaloop FSPIOP API Definition.
+
+### UUID
+
+For details, see section [7.2.16 UUID](../fspiop-api/documents/API-Definition_v1.1.md#7216-uuid) in the Mojaloop FSPIOP API Definition.
+
+## Element definitions
+
+This section defines element types used by the API.
+
+### AutoPositionReset

```

Comment:
Why does this need a separate definition? It's already defined inline in the above data models.

Reply:
Good point, will remove it.